src/components/Navigation/Navigation.js

```diff
@@ -1,7 +1,10 @@
 import React, { useState, useEffect } from 'react'
 import classNames from 'classnames'
-import AssignmentTurnedInIcon from '@material-ui/icons/AssignmentTurnedIn'
-import AttachMoneyIcon from '@material-ui/icons/AttachMoney'
+
```

Comment:
I added this line break so the rest can be sorted easily without these moving around

Reply:
Good call, I like this `import` organization pattern!